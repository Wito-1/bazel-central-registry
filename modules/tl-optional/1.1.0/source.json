{
    "url": "https://github.com/TartanLlama/optional/archive/refs/tags/v1.1.0.tar.gz",
    "integrity": "sha256-iOznnz3lzK7EGRlRoiL5XMgMQ4Ha/TFjvbH/h87fMRg=",
    "strip_prefix": "optional-1.1.0",
    "patch_strip": 0,
    "overlay": {
        "BUILD.bazel": "sha256-uWrIZhZeuf5Db+oKsH3J/eDUarD4Rh2DLPlcy/ssCa8=",
        "MODULE.bazel": "sha256-7OQKwUBb2vHbc054yRqzVKMVd5P/bE7VWk83yuGSefg="
    }
}
