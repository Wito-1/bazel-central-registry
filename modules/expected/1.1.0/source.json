{
    "url": "https://github.com/TartanLlama/expected/archive/refs/tags/v1.1.0.tar.gz",
    "integrity": "sha256-HbNX9G3SskRHFWqvlwxMQKeT7xKoqcKtngltmAE2jfY=",
    "strip_prefix": "expected-1.1.0",
    "patch_strip": 0,
    "overlay": {
        "BUILD.bazel": "sha256-tPPfIBtgrE3G9AXLvl4rsLfBYKxqiKqkVUqVF5gXdUc=",
        "MODULE.bazel": "sha256-q0OQ70XH2Z8blFfozL6tR4nrGFXhV1cucoze46VfsZI="
    }
}
